Source/Core/Core/HotkeyManager.cpp

```diff
@@ -354,7 +356,8 @@ constexpr std::array<HotkeyGroupInfo, NUM_HOTKEY_GROUPS> s_groups_info = {
      {_trans("Other State Hotkeys"), HK_SAVE_FIRST_STATE, HK_DECREMENT_SELECTED_STATE_SLOT},
      {_trans("GBA Core"), HK_GBA_LOAD, HK_GBA_RESET, true},
      {_trans("GBA Volume"), HK_GBA_VOLUME_DOWN, HK_GBA_TOGGLE_MUTE, true},
-     {_trans("GBA Window Size"), HK_GBA_1X, HK_GBA_4X, true}}};
+     {_trans("GBA Window Size"), HK_GBA_1X, HK_SKYLANDERS_PORTAL, true},
```

Comment:
This looks wrong, should probably remain `HK_GBA_4X`.

Reply:
Agreed. Forgot to change it back